corehq/apps/reminders/management/commands/migrate_to_new_reminders.py

```diff
@@ -498,6 +499,11 @@ def handle(self, domain, **options):
         if not check_only:
             self.ensure_migration_flag_enabled(domain)
 
+        if not check_only and couch_sql_migration_in_progress(domain):
```

Comment:
@jmtroth0 if you wanted to check for the reminders migration in your script as well you could just check [REMINDERS_MIGRATION_IN_PROGRESS.enabled(domain)](https://github.com/dimagi/commcare-hq/blob/af69cf1be9d1d4372b21a9e26467b09545fe45f8/corehq/toggles.py#L1146)

Reply:
Thanks! https://github.com/dimagi/commcare-hq/pull/20711